### DISCUSSION TITLE
When trying to access the control panel traffic - 404 page not found [Pterodactyl + traefik + bind9]

### DISCUSSION THREAD

**Discussion by R1venDev** [pterodactyl/panel#4917] [Support]
Hi all. I've had a problem that. Using the manual [https://technotim.live/posts/pterodactyl-game-server/#reverse-proxy] and [https://github.com/EdyTheCow/docker-pterodactyl] I installed pterodactyl, traefik and bind9.

Important: I installed traefik, bind9 and pterodactyl using docker.

Important: before we start, it should be said that ABSOLUTELY ALL the tests I did were done exclusively on a local server, from which I tried to access the pterodactyl control panel. The ubuntu firewall (UFW) was completely disabled. I was not trying to access my server from another computer, this is important.

So, let's start with the bundle I'm using and why.
I use CloudFlare for domain and ssl certificate issuance via traefik.
I have configured the dns records as follows. Since I have a global domain example.pro - I created a global subdomain: local.example.pro, which redirected to the ip address of the host machine. 
On the host machine itself, I raised bind9 with the docker, in which I configured the following DNS entries [Screenshot 1]
As you can see there is a local panel subdomain on the host machine itself, which is used by both pterodactyl and traefik. Important: in the case of the traefik control panel - everything works fine.

All subdomains are configured correctly and working, this is confirmed by the nslookup test I have attached in [second screenshot]. There are NO questions in the correct configuration of bind9.

We're done with bind, now let's move on to traefik, here I'll just attach some screenshots. And I will say the following: all ssl certificates are issued and everything works.
Traefik docker-compose.yml [Screenshot 3]
Traefik config.yml [Screenshot 4]
Traefik traefik.yml [Screenshot 5]
Traefik dashboacrd [screenshot 6]
Traefik ssl cert *.panel.local.example.pro [Screenshot 7]

And now we come to the most important thing, the thing that's been causing me a soul-cry for the past two weeks - the pterodactyl panel.
I have everything set up, but when I try to log into the pterodactyl control panel I see: 404 page not found. I had a similar problem on traefik, then I solved it by installing whoami web server, but now I have no idea what to do. I don't see any errors from my side, so please help me.
In the screenshots:
8 - I have attached the docker-compos.yml file
9 - attached is a 404 error, when trying to log in to the control panel.
10 - curl request that addresses me to the correct ip, which shows that everything is set up perfectly on the bind9 and traefik side from the traffic routing point of view.
11 - docker logs pterodactyl panel, which inform me about some error in issuing ssl, but ssl is issued and I have no doubts that ssl is to blame, I think that ssl has nothing to do with it.

Bottom line: I have traefik, bind9 (as local dns) and pterodactyl configured, however the traefik and ssl control panel login works correctly, but the pterodactyl panel does not. Any hints and help would be appreciated.

Discord (pterodactyl) topic: https://discord.com/channels/122900397965705216/1172055462737362974

pterodactyl panel docker logs: 

https://pastebin.com/Tew6Vr1v

![1](https://github.com/pterodactyl/panel/assets/104576430/a7cf516c-f852-4d1c-b9f5-b037ec6d9035)
![2](https://github.com/pterodactyl/panel/assets/104576430/99faf367-a23f-4191-8bb8-937d8d6244ac)
![3](https://github.com/pterodactyl/panel/assets/104576430/dd04d45e-457a-463a-9c86-d142564147c8)
![4](https://github.com/pterodactyl/panel/assets/104576430/03f4e038-9ede-4690-8f65-eff9d2663aa6)
![5](https://github.com/pterodactyl/panel/assets/104576430/e904110e-a222-4bf4-b607-2c9ac83f66a2)
![6](https://github.com/pterodactyl/panel/assets/104576430/e1f0937d-5edf-415e-a65d-6ccd0fa15be3)
![7](https://github.com/pterodactyl/panel/assets/104576430/6084be39-e895-4a26-8531-b9078ac678f7)
![8](https://github.com/pterodactyl/panel/assets/104576430/9557b622-ce4a-4485-be9f-f766b0888f92)
![9](https://github.com/pterodactyl/panel/assets/104576430/1f0094e2-4bfb-45d2-94d4-02dd86a66bbe)
![10](https://github.com/pterodactyl/panel/assets/104576430/a1be52d0-bb16-4ced-a054-2700da428852)
![11](https://github.com/pterodactyl/panel/assets/104576430/cffedf8a-d644-40fc-bcf6-9924dfffddf7)
![12](https://github.com/pterodactyl/panel/assets/104576430/ead4f407-d14b-44cd-aa72-2b48e5bc3f07)
![13](https://github.com/pterodactyl/panel/assets/104576430/8d367738-5f2a-46d8-ac59-7bccfe8ebe93)

**Comment by mylogon341** [comment]
@R1venDev did you ever find the solution to this? I'm experiencing the same problem, is all.
thanks
